index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="styles.css" />
+    <title>Andres -- Martin</title>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="main" class="main">
+      <h1 id="title">Nombre Blog</h1>
+      <p>mensaje titulo</p>
+      <figure id="img-div">
+          <div>
+              <img id="image" src="img/computador.jpg"  />
+              <figcaption id="img-caption">
+                  "Infografia de la imagen"
```

Comment:
Kindly consider to remove this unnecessary double quotes..

Reply:
<b>Changes required</b>
I noticed some unnecessary double quotes in the figcaption element of your HTML code. While this may not cause any technical issues, it can make the text appear unprofessional and inconsistent with coding standards and best practices. In HTML, it's generally recommended to use double quotes to enclose attribute values, and single quotes for other purposes, such as enclosing JavaScript strings. Using unnecessary double quotes in HTML markup can make the code harder to read and maintain, and it may cause confusion for other developers who review the code. To make your code more consistent and easier to read and maintain, I would suggest removing the unnecessary double quotes and using single quotes instead.